.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+apollo.config.js
```

Comment:
What is this file used for?

Reply:
Apollo has tools for working on projects using their framework. The tools specific for Apollo Client need that configuration file. In my case I'm using an extension for VSCode which offers code completion for GraphQL queries against an Apollo Server.

I was in doubt about adding that file to `.gitignore`, but I also didn't want to commit the file accidentally.